packages/backend/src/filter.ts

```diff
@@ -32,17 +32,26 @@ function getTypes(filterObject?: any): string[] {
   return _.compact(_.map(types, _.trim));
 }
 
+function getHideFromWizard(filterObject?: any): boolean {
+  return _.get(filterObject, "hideFromWizard", false);
+}
+
 export class GeneratorFilter {
   public static create(filterObject?: any) {
     const categories: string[] = getCategories(filterObject);
     const types: string[] = getTypes(filterObject);
+    const hideFromWizard: boolean = getHideFromWizard(filterObject);
 
-    return new GeneratorFilter(types, categories);
+    return new GeneratorFilter(types, categories, hideFromWizard);
```

Comment:
please consider to use the existing filters mechanism and create an additional category, for example, 'hidden'. if it will serve your aims you should do nothing changes in GeneratorFilter constructor rather apply the category filters more accuratelly.

Reply:
done, thanks